src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -225,8 +225,9 @@ export default class AmalgamatingBusinesses extends Mixins(CommonMixin) {
 
     // Set the amalgamated businesses array in the store.
     this.setAmalgamatingBusinesses(this.amalgamatingBusinesses)
+
     // Close the "Add an Amalgamating Business" Panel.
-    this.addAmalgamatingBusinessCancel()
+    this.isAddingAmalgamatingBusiness = false
```

Comment:
Up to you. If you think "cancel" might do more work then I'll revert this, otherwise I prefer no redirection. lmk